obs_scm_bridge

```diff
@@ -283,6 +283,8 @@ class ObsGit(object):
                     continue
                 path = gsmconfig[section]['path']
                 url = gsmconfig[section]['url']
+                if url.startswith('../'):
+                    url = urllib.parse.urljoin(urllib.parse.urlunparse(self.url), url)
```

Comment:
This does not handle urls on the file system correctly, you have to append a "/" to the url for that:
```suggestion
                    url = urllib.parse.urljoin(urllib.parse.urlunparse(self.url) + "/", url)
```